Zhana_Konsulova_7MI0800248_group6/Prakt2_zad2/Prakt2_zad2/Prakt2_zad2.cpp

```diff
@@ -0,0 +1,90 @@
+// Prakt2_zad2.cpp : This file contains the 'main' function. Program execution begins and ends there.
+//
+
+#include <iostream>
+
+
+struct Pair
+{
+    int first, second;
+    void Read()
+    {
+        std::cin >> first >> second;
+    }
+    void Print() const
+    {
+        std::cout << "(" << first << ", " << second << ")" << std::endl;
+    }
+};
+
+bool IsFirstPairBigger(const Pair& p1,const Pair& p2)
```

Comment:
A typical pattern in programming is to use a function called `compare` which takes two elements of the same type (`left` & `right`) and returns `a negative int` if `left is smaller than right`, `zero` if `left is equal to right`, `a positive int` if `left is greater than right`.